Update delete-reset-tenant.md

tamigoodall opened the pull request:
Added note
Restructured article to move text to appropriate sections
Removed outdated screenshots
Added related topics section
Minor changes to related topics for consistency

https://docs-content-staging-pr-7952.herokuapp.com/docs/support/delete-reset-tenant

<!---
Notes:
- Your PR should conform to our [Contributing Guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md)
- If applicable, add details to the [update feed](https://github.com/auth0/docs/tree/master/updates)
- Make sure your PR gets deployed to a Heroku [Review App](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#review-apps) without errors
- Please include a short description
- If your PR is a work in progress, title it [DO NOT MERGE]
--->

tamigoodall committed:
Update delete-reset-tenant.md

Added note

tamigoodall commented:
Submitted request for review: https://auth0team.atlassian.net/browse/ESD-1159

solepano commented on articles/support/delete-reset-tenant.md:
```diff
@@ -16,6 +15,12 @@ useCase:
 
 Using the [Reset Tenant Extension](https://github.com/auth0-extensions/auth0-reset-tenant), you can programmatically make changes to your Auth0 tenant. Not only is this useful for creating new environments for development or test purposes, you can use the extension to "clean" your tenant and remove unwanted items.
 
+::: note
```
This title is about delete vs reset tenant. I'd put this note under the "Delete your Auth0 Tenant" title instead, after the warning that deletion cannot be restored.

solepano replied:
Also for that warning, instead of "This action cannot be undone." I'd say "Deleted tenants cannot be restored and the tenant name may not be used again when creating new tenants." Same as in here https://auth0.com/docs/policies/restore-deleted-tenant.

tamigoodall replied:
I'm rewriting the entire article to move notes and warnings to appropriate sections.

tamigoodall committed:
Reworked whole topic

Reworked entire topic to move text to appropriate sections

tamigoodall committed:
Added keep reading list

Added Keep reading list of related articles
Made minor changes to related articles for consistency

tamigoodall committed:
Update _delete-tenant.md

Removed out of date screenshot

solepano reviewed (dismissed):
👍

khorne3 commented on articles/support/_delete-tenant.md:
```diff
@@ -1,21 +1,21 @@
-1. Log in to the [dashboard](${manage_url}).
-2. Click on your username in the top right corner to bring up the associated dropdown box.
+::: panel Enterprise customers
+If you are an Enterprise customer, you may have multiple child tenants as part of your Auth0 contract. Rest assured that if you choose to delete such a tenant, this will have no impact on your agreement with Auth0. We recommend that if you have any concerns about your Enterprise agreement, then discuss them with your Customer Success Manager (CSM).
```
```suggestion
If you are an Enterprise customer, you may have multiple child tenants as part of your Auth0 contract. Rest assured that if you choose to delete such a tenant, this will have no impact on your agreement with Auth0. We recommend speaking with your Customer Success Manager (CSM) if you have any concerns about your Enterprise agreement.
```

khorne3 commented on articles/support/removing-auth0-exporting-data.md:
```diff
@@ -11,21 +11,20 @@ useCase:
+# Export Data Out of Auth0
 
 All data in your Auth0 tenant is always under your control and is [available through the management API](/api/v2) at any time.
 The only information which is not available through the API are the password hashes of your [Auth0-hosted database users](/connections/database) and private keys, for security reasons.
```
```suggestion
The only information which is not available through the API (for security reasons) are the password hashes of your [Auth0-hosted database users](/connections/database) and private keys.
```

tamigoodall committed:
Update articles/support/_delete-tenant.md

Co-Authored-By: K Horne <katie@auth0.com>

tamigoodall committed:
Update articles/support/removing-auth0-exporting-data.md

Co-Authored-By: K Horne <katie@auth0.com>
